Create ui-c8y-1023-78-5-added-softwaretype-to-be-part-of-the-managed-object-in-a.md (#4672)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1023-78-5-added-softwaretype-to-be-part-of-the-managed-object-in-a.md

---
date: ""

> title: Device profile now always includes software type in the managed object

Reviewer: ```suggestion
title: Device profile assignment now includes software type in operations
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5994
version: 1023.78.5
---

> When creating or updating a device profile, the software type information was not consistently included in the managed object data structure, which could cause issues when retrieving or managing device profiles. The device profile now always includes the software type as part of the managed object, ensuring that all software type information is properly stored and available when you access device profiles. This change ensures consistency in how device profiles are managed and prevents data loss or incomplete profile information when working with devices that have software type configurations.

Reviewer: ```suggestion
When assigning a device profile, the software type defined in the profile was not included in the generated operation.
The operation now includes the software type when it is defined in the profile, ensuring correct profile application on devices.
```

---
date: ""
title: Device profile assignment now includes software type in operations
product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5994
version: 1023.78.5
---

> When assigning a device profile, the software type defined in the profile was not included in the generated operation.

Reviewer 2: ```suggestion
When assigning a device profile, the software type defined in the profile was not included in the generated operation. This issue has been fixed.
```

> The operation now includes the software type when it is defined in the profile, ensuring correct profile application on devices.

Reviewer 2: application > assignment?

Reviewer 3: Nope, this is truly about the device profile being locally applied on a device. The wording seems correct to me.
